src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="html"
+>
```

Comment:
The attributes of the `<html>` tag should be indented with 2 spaces. The closing bracket should be aligned with the opening tag.